indico/htdocs/js/indico/jquery/qbubble.js

```diff
@@ -41,8 +41,29 @@
             var self = this;
             var classes = self.options.style ? self.options.style.classes : '';
 
+            var originalHideCallback = self.options.events && self.options.events.hide;
+            var originalFocusCallback = self.options.events && self.options.events.focus;
+            var hideCallback = function(evt, api) {
```

Comment:
you could just use `function hideCallback` - no need for the `var hideCallback = ...` notation